migrations/20170622165827-ports-migration.js

```diff
@@ -0,0 +1,130 @@
+module.exports = {
```

Comment:
ahh so this is named as such, because it will only be ran once. In the future if we do another migration, we will name it with a similar structure.